### PR TITLE
Saba/home

### PR DESCRIPTION
**Pull request by SabaSaleem5317**
I developed a home page that contains following sections:

1. **About Us
2. Gallery
3. Achievements
4. Contact Us:** 

image

In addition to these sections, I integrated  chatbot feature into the home page. The chatbot can be accessed through an option in the navbar. When a user clicks on this option, a modal appears on the screen, presenting the chatbot interface. 
image

### CODE REVIEW

**Review by anssidhu — APPROVED** [review]
Looks good! I hope the slider is working properly now.

**Review by azubair-experlabs — APPROVED** [review]
Looks good now (though the name of the branch could be improved)

**Comment by SabaSaleem5317** [comment]
> Looks good now (though the name of the branch could be improved)

ok I'll improve it next time.

**Review by muhammadqasim21 — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -3,14 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.5.2",
+    "@fortawesome/react-fontawesome": "^0.2.2",
```
Comment (UrwaIhtesham) [inline frontend/package.json:7]: Very Good work

File: frontend/public/assets/pics/budgie.PNG
Comment (UrwaIhtesham) [inline frontend/public/assets/pics/budgie.PNG:1]: cute

Comment (SabaSaleem5317) [reply frontend/public/assets/pics/budgie.PNG:1]: very much

**Review by UrwaIhtesham — APPROVED** [review]

**Pull request merged**